Explorer++/ShellBrowser/SortManager.cpp

```diff
@@ -335,8 +335,14 @@ int CALLBACK CShellBrowser::Sort(int InternalIndex1,int InternalIndex2) const
 	{
 		/* By default, items that are equal will be sub-sorted
 		by their display names. */
-		ComparisonResult = StrCmpLogicalW(m_pExtraItemInfo[InternalIndex1].szDisplayName,
-			m_pExtraItemInfo[InternalIndex2].szDisplayName);
+		if (m_bSortNonLogical) {

```

Comment:
The opening brace should be on the next line (same with the else statement below).